src/benchmarklib/tpch/tpch_benchmark_item_runner.cpp

```diff
@@ -71,7 +71,6 @@ void TPCHBenchmarkItemRunner::on_tables_loaded() {
   // Make sure that sort order, indexes, and constraints have made it all the way up to here
   const auto orders_table = Hyrise::get().storage_manager.get_table("orders");
   const auto first_chunk = orders_table->get_chunk(ChunkID{0});
-  Assert(!first_chunk->individually_sorted_by().empty(), "Sorting information was lost");
   if (_config->indexes) {
```

Comment:
Not sure how to get this check back into the item runner without passing the enum. Opinions?

Reply:
I'm risking a new discussion here: Should we pass the information that the default-generated data is also sorted by certain key columns into the system?

Reply:
I'd say no, because we actually do not have this knowledge. It could be quite helpful, but without having the encoder/loading automatically checking for this property (which would again be quite nice and should more or less come for free), I'd say no.

Reply:
> I'd say no, because we actually do not have this knowledge. It could be quite helpful, but without having the encoder/loading automatically checking for this property (which would again be quite nice and should more or less come for free), I'd say no.

No strong opinion, but I agree with you